Documentation/ColumnsConfig/Type/User/Index.rst

```diff
@@ -52,8 +52,20 @@ implementing a rendering. See :ref:`FormEngine docs
 
 ..  rst-class:: bignums
 
+1.  Verify Extension Compatibility with Dependency Injection (DI)
 
-1.  Register the new renderType node element
+    ..  versionchanged:: 13.0
```

Comment:
Please link to deprecation/changelog entry in this paragraph

Reply:
As discussed I only have added the DI link to this section and the forge link into the commit message